dpnp/backend/extensions/lapack/gesv.hpp

```diff
@@ -39,11 +39,17 @@ namespace ext
 namespace lapack
 {
 extern std::pair<sycl::event, sycl::event>
-    gesv(sycl::queue exec_q,
+    gesv(sycl::queue &exec_q,
          dpctl::tensor::usm_ndarray coeff_matrix,
          dpctl::tensor::usm_ndarray dependent_vals,
          const std::vector<sycl::event> &depends);
 
+extern std::pair<sycl::event, sycl::event>
+    gesv_batch(sycl::queue &exec_q,
+               dpctl::tensor::usm_ndarray coeff_matrix,
```

Comment:
Can it be const and passed by a reference?

Reply:
We can not pass `exec_q` as a constant argument because `mkl::lapack::gesv` expects queue as a regular argument
```
error: no matching function for call to 'gesv'
  145 |             gesv_event = mkl_lapack::gesv(
      |                          ^~~~~~~~~~~~~~~~
miniforge3/envs/test/include/oneapi/mkl/lapack/lapack.hpp:68:24: note: candidate function not viable: 1st argument ('const sycl::queue') would lose const qualifier
```